docs/pulseload.md

```diff
@@ -136,18 +131,12 @@ The example settings above would produce the following settings:
         "try",
         "mozilla-central",
     ],
-    "destinations": [
```

Comment:
The file ``submitting_data.md`` also has a couple references to ``destinations`` that should be removed.

Reply:
Good catch!  I've fixed and rebased into the commit :)